sdk/arm/keyvault/2019-09-01/armkeyvault/example_vaults_test.go

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package armkeyvault_test
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/arm/keyvault/2019-09-01/armkeyvault"
+	"github.com/Azure/azure-sdk-for-go/sdk/armcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/to"
+)
+
+func ExampleVaultsOperations_BeginCreateOrUpdate() {
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		log.Fatalf("failed to obtain a credential: %v", err)
+	}
+	client := armkeyvault.NewVaultsClient(armcore.NewDefaultConnection(cred, nil), "<subscription ID>")
+	poll, err := client.BeginCreateOrUpdate(
+		context.Background(),
+		"<resource group name>",
+		"<vault name>",
+		armkeyvault.VaultCreateOrUpdateParameters{
+			Location: to.StringPtr("<Azure location>"),
+			Properties: &armkeyvault.VaultProperties{
+				TenantID: to.StringPtr("<tenant ID>"),
+				SKU: &armkeyvault.SKU{
+					Family: armkeyvault.SKUFamilyA.ToPtr(),
+					Name:   armkeyvault.SKUNameStandard.ToPtr(),
+				},
+				AccessPolicies: &[]armkeyvault.AccessPolicyEntry{},
+			},
+		}, nil)
+	if err != nil {
+		log.Fatalf("failed to obtain a response: %v", err)
+	}
+	resp, err := poll.PollUntilDone(context.Background(), 5*time.Second)
+	if err != nil {
+		log.Fatalf("failed to obtain a response: %v", err)
```

Comment:
I don' think this really failed to obtain a response, it's more that the operation failed.

Reply:
Looks to be copy/paste in all the samples.

Reply:
Done. Updated the error output messages.